src/model.cpp

```diff
@@ -373,10 +393,16 @@ Model::load_3ds(const std::string &filename)
     while (!input_file.eof()) {
         uint16_t chunk_id;
         uint32_t chunk_length;
-
+		uint32_t swap32t;
```

Comment:
glmark2 uses 4 spaces for indentation (not tabs). Also applies to all code below.